test/integration/scep/common_test.go

```diff
@@ -39,6 +39,9 @@ func newCAClient(t *testing.T, caURL, rootFilepath string) *ca.Client {
 }
 
 func requireHealthyCA(t *testing.T, caClient *ca.Client) {
+	// Wait for CA
+	time.Sleep(time.Second)
```

Comment:
I see I forgot this before:
```suggestion
func requireHealthyCA(t *testing.T, caClient *ca.Client) {
        t.Helper()
	// Wait for CA
	time.Sleep(time.Second)
```